samples/ConsoleApp/Program.cs

```diff
@@ -1,46 +1,47 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using Microsoft.DurableTask;
-using Microsoft.DurableTask.Grpc;
-using Microsoft.Extensions.Logging;
-
-ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
-{
-    builder.SetMinimumLevel(LogLevel.Debug);
-    builder.AddSimpleConsole(options =>
-    {
-        options.SingleLine = true;
-        options.UseUtcTimestamp = true;
-        options.TimestampFormat = "yyyy-mm-ddThh:mm:ss.ffffffZ ";
-    });
-});
-
-DurableTaskGrpcWorker worker = DurableTaskGrpcWorker.CreateBuilder()
-    .AddTasks(tasks =>
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Worker;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices(services =>
     {
-        tasks.AddOrchestrator("HelloSequence", async context =>
+        services.AddDurableTaskClient(builder =>
         {
-            var greetings = new List<string>
-            {
-                await context.CallActivityAsync<string>("SayHello", "Tokyo"),
-                await context.CallActivityAsync<string>("SayHello", "London"),
-                await context.CallActivityAsync<string>("SayHello", "Seattle"),
-            };
+            builder.UseGrpc();
+            builder.RegisterDirectly();
```

Comment:
This line registers the `DurableTaskClient` directly to the service collection. I am wondering if we should register the "Default" (no name provided) client to the service container?

Reply:
I'm afraid I don't understand the type architecture well enough to understand the question. I also worry customers might struggle to understand like I am. The call to `host.Services.GetRequiredService<DurableTaskClient>()` makes me a tiny bit uneasy since it's not particularly discoverable, but I think it's okay. I do worry about this `builder.RegisterDirectly()` requirement though. I would never think to call that. Could be make this the default configuration?

Reply:
#81 partially addresses this - removing the need for `RegisterDirectly` in this case. I will update this PR to reflect that.

Reply:
Are we able to remove this now?